example/src/subdir/gitlab-generated.js

```diff
@@ -0,0 +1,2 @@
+// Ignored by https://github.com/aspect-build/rules_lint/blob/example/.gitattributes
+console.log("This is an intentional lint error");
```

Comment:
it's a format error, not lint right?

Reply:
No, the console.log is made to be a lint error because `no-console` is enabled in .eslintrc

Reply:
I've updated the string to make that more clear hopefully

Reply:
But the ignore handling you've added is only for the formatter, which is unrelated to eslint. So this can't possibly be testing the feature